[2022-12-13T16:25:55.042+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:25:53.048979+00:00 [queued]>
[2022-12-13T16:25:55.052+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:25:53.048979+00:00 [queued]>
[2022-12-13T16:25:55.053+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-13T16:25:55.053+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-13T16:25:55.053+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-13T16:25:55.079+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): etl_bus_loc> on 2022-12-13 16:25:53.048979+00:00
[2022-12-13T16:25:55.086+0000] {standard_task_runner.py:55} INFO - Started process 749 to run task
[2022-12-13T16:25:55.089+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bus_pos_etl', 'etl_bus_loc', 'manual__2022-12-13T16:25:53.048979+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/test_noe_bash.py', '--cfg-path', '/tmp/tmp2zpvx8m7']
[2022-12-13T16:25:55.090+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask etl_bus_loc
[2022-12-13T16:25:55.150+0000] {task_command.py:389} INFO - Running <TaskInstance: bus_pos_etl.etl_bus_loc manual__2022-12-13T16:25:53.048979+00:00 [running]> on host ff6f332d28b5
[2022-12-13T16:25:55.219+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=noe
AIRFLOW_CTX_DAG_ID=bus_pos_etl
AIRFLOW_CTX_TASK_ID=etl_bus_loc
AIRFLOW_CTX_EXECUTION_DATE=2022-12-13T16:25:53.048979+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-13T16:25:53.048979+00:00
[2022-12-13T16:25:55.220+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-13T16:25:55.221+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/etl/etl.py']
[2022-12-13T16:25:55.235+0000] {subprocess.py:86} INFO - Output:
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO -   File "/opt/***/dags/etl/etl.py", line 9, in <module>
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO -     storage_client = storage.Client.from_service_account_json('/root/midyear-cursor-371415-a5bbee084fb8.json')
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO -   File "/home/***/.local/lib/python3.7/site-packages/google/cloud/client/__init__.py", line 106, in from_service_account_json
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO -     with io.open(json_credentials_path, "r", encoding="utf-8") as json_fi:
[2022-12-13T16:25:56.224+0000] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: '/root/midyear-cursor-371415-a5bbee084fb8.json'
[2022-12-13T16:25:56.340+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2022-12-13T16:25:56.353+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-12-13T16:25:56.357+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=bus_pos_etl, task_id=etl_bus_loc, execution_date=20221213T162553, start_date=20221213T162555, end_date=20221213T162556
[2022-12-13T16:25:56.376+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 14 for task etl_bus_loc (Bash command failed. The command returned a non-zero exit code 1.; 749)
[2022-12-13T16:25:56.386+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-13T16:25:56.413+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
